### PR TITLE
Issue 165

### PR DESCRIPTION
**Pull request by sanchitanec**
system Monitoring update

### CODE REVIEW

**Review by showersky — APPROVED** [review]

File: doc/en/source/system_monitoring.rst [hunk 1]
```diff
@@ -13,144 +13,106 @@ As illustrated by the following picture, in order to set up FogFlow System Monit
...
+Fetch all required scripts
+-------------------------------------------------------------
 
+Download the docker-compose file and the configuration files as below.
 
-- Change the details of Elasticsearch in metricbeat.docker.yml file as below:
+.. code-block:: console    
 
+	# the docker-compose file to start all Monitoring components on the cloud node
+	wget https://raw.githubusercontent.com/smartfog/fogflow/master/docker/core/http/grafana/docker-compose.yml
```
[the remark this answers is not in the document]
Comment (sanchitanec) [reply doc/en/source/system_monitoring.rst:28]: This path is as per the tutorial, ( tutorial reflects github files from master branch). 
When pull request will merge into master branch this path will be accessible. So no changes made

File: doc/en/source/system_monitoring.rst [hunk 2]
```diff
@@ -13,144 +13,106 @@ As illustrated by the following picture, in order to set up FogFlow System Monit
...
+	
+	# the configuration file used by Grafana
+	wget https://raw.githubusercontent.com/smartfog/fogflow/master/docker/core/http/grafana/grafana.yaml
```
[the remark this answers is not in the document]
Comment (sanchitanec) [reply doc/en/source/system_monitoring.rst:31]: This path is as per the tutorial, ( tutorial reflects github files from master branch). 
When pull request will merge into master branch this path will be accessible. So no changes made

File: doc/en/source/system_monitoring.rst [hunk 3]
```diff
@@ -13,144 +13,106 @@ As illustrated by the following picture, in order to set up FogFlow System Monit
...
-          metricsets: ["cpu","memory","network"]
-          hosts: ["unix:///var/run/docker.sock"]
-          period: 10s
-          enabled: true
-        - module: system
-          #System module parameters that has to be monitored based on user requirement, example as below
-          metricsets: ["cpu","load","memory","network"]
-          period: 10s
+        # JSON files to configure grafana dashboard 
+	wget https://raw.githubusercontent.com/smartfog/fogflow/master/docker/core/http/grafana/dashboards
```
[the remark this answers is not in the document]
Comment (sanchitanec) [reply doc/en/source/system_monitoring.rst:37]: created zip file for the dashboard folder

File: doc/en/source/system_monitoring.rst [hunk 4]
```diff
@@ -13,144 +13,106 @@ As illustrated by the following picture, in order to set up FogFlow System Monit
...
-	# the configuration file used by the nginx proxy
-	wget https://raw.githubusercontent.com/smartfog/fogflow/master/docker/core/http/nginx.conf
+Grafana dashboard can be accessible on web browser to see the current system status via the URL: http://<output.elasticsearch.hosts>:3003/. 
+The default username and password for Grafana login are admin and admin respectively.
 
-        # the configuration file used by metricbeat
-        wget https://raw.githubusercontent.com/smartfog/fogflow/master/docker/core/http/metricbeat.docker.yml
 
+- After successful login to grafana, click on "Add data source" on Home Dashboard to setup the source of data.
+- Select Elasticsearch from Add Data Sourch page. Now the new page is Data Sources/Elasticsearch same as below figure.
```
[the remark this answers is not in the document]
Comment (sanchitanec) [reply doc/en/source/system_monitoring.rst:66]: done

File: doc/en/source/system_monitoring.rst [hunk 5]
```diff
@@ -13,144 +13,106 @@ As illustrated by the following picture, in order to set up FogFlow System Monit
...
 Grafana-based monitoring
 ===========================================================  
         
-To monitor metrics of FogFlow cloud as well as edge nodes in graphical format we need to setup dashboard.
-Here are some basic Grafana visualization dashboard setting examples to monitor system resources.
+Grafana based system metrics can be seen from Home Dashboard page of Grafana. In the sidebar, take the cursor over Dashboards (squares) icon, and then click Manage. The dashboard appears in a Services folder.
+
+Here are some basic Grafana visualization dashboard to monitor metrics of FogFlow cloud as well as edge nodes.
 
 - **Below diagram illustrate steps to setup dashboard for containers list with maximum memory usage**.
```
[the remark this answers is not in the document]
Comment (sanchitanec) [reply doc/en/source/system_monitoring.rst:92]: done

**Pull request merged**